plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
+                        required=True, help="Label Selector")
+    return parser
+
+
+check_mongodb_cmd = ("bash", "-c",
+                     "mongo 127.0.0.1/admin "
+                     "-u \"admin\" -p \"$MONGODB_ADMIN_PASSWORD\" "
+                     "--eval=\"rs.status().myState\" --quiet")
+
+
+def parse_mongo_result(output):
+    try:
+        return int(output.rstrip("\n"))
+    except:
+        return None
+
+
+def analize(status):
+    if status is None:
+        return nagios.UNKNOWN
+    elif status == ReplsetStatuses.PRIMARY or status == ReplsetStatuses.SECONDARY:
+        return nagios.OK
+    elif status == ReplsetStatuses.ARBITER or status == ReplsetStatuses.STARTUP:
+        return nagios.WARN
+    else:
+        return nagios.CRIT
+
+
+def report(pods, rs_statuses, nag_statuses):
+    if len(filter(None, pods)) < 1:
```

Comment:
And when you say `< 1` it really means `== 0`, since length is never negative :)

And since we're saying `== 0`, the pythonic way is to simply say:

```python
if not pods:
    ...
```